### PR TITLE
Adding Danish translations to Wallabag v2

### PR DESCRIPTION
**Pull request by mjjzf**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | yes
| Fixed tickets | no
| License       | MIT

### CODE REVIEW

**Comment by mjjzf** [comment]
Had to fix indentation on config.da.yml. Now done.

File: src/Wallabag/CoreBundle/Resources/translations/messages.da.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+Mark as read: 'Marker som læst'
+Favorite: 'Marker som favorit'
+back: 'tilbage'
+original article: 'Originalartikel'
+Add a tag: 'Tliføj et tag'
+Share: 'Deling'
+Download: 'Download'
+Does this article appear wrong?: "Vises artiklen forkert?"
+Problems?: 'Problemer?'
+Edit title: "Rediger titel"
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.da.yml]: Could you please add an empty line at the end of the file?

**Comment by mjjzf** [comment]
Added additional newline as requested in messages.da.yml.

File: src/Wallabag/CoreBundle/Resources/translations/messages.da.yml (hunk 1 above)
Comment (mjjzf) [inline src/Wallabag/CoreBundle/Resources/translations/messages.da.yml]: Added additional newline as requested in messages.da.yml.

**Comment by j0k3r** [comment]
Great, thanks ! :+1:

**Pull request merged**